### PR TITLE
Handle inotifyBase FileNotFoundError.

### PR DESCRIPTION
**Pull request by ahmubashshir**
Sometimes, a process ends before we open ther /proc/<pid> directory and open() throws a FileNotFoundError.
If this excpetion is raised, we will silently ignore the pid and continue the loop.

### CODE REVIEW

File: trackma/tracker/inotifyBase.py [hunk 1]
```diff
@@ -42,10 +42,12 @@ def _proc_poll(self):
...
-            with open('/proc/%s/cmdline' % p, 'rb') as f:
-                cmdline = f.read()
-                pname = cmdline.partition(b'\x00')[0]
-
+            try:
+		        with open('/proc/%s/cmdline' % p, 'rb') as f:
+		            cmdline = f.read()
+		            pname = cmdline.partition(b'\x00')[0]
+			except FileNotFoundError:
+				continue			# Continue without raising exception if process doesn't exist.
```
Comment (FichteFoll) [inline trackma/tracker/inotifyBase.py:50]: Something went terribly wrong with the indentation here. It seems that you inserted tabs when the file is using spaces.

Comment (ahmubashshir) [reply trackma/tracker/inotifyBase.py:50]: Sorry...

Comment (ahmubashshir) [reply trackma/tracker/inotifyBase.py]: Fixed :smile:

File: trackma/tracker/inotifyBase.py [hunk 2]
```diff
@@ -42,10 +42,12 @@ def _proc_poll(self):
...
-            with open('/proc/%s/cmdline' % p, 'rb') as f:
-                cmdline = f.read()
-                pname = cmdline.partition(b'\x00')[0]
-
+            try:
+                with open('/proc/%s/cmdline' % p, 'rb') as f:
+                    cmdline = f.read()
+                    pname = cmdline.partition(b'\x00')[0]
+            except FileNotFoundError:
+                continue            # Continue without raising exception if process doesn't exist.
```
Comment (FichteFoll) [inline trackma/tracker/inotifyBase.py:50]: This comment is entirely redundant. You can see everything it describes by just reading the code.

Comment (ahmubashshir) [reply trackma/tracker/inotifyBase.py]: I've removed the comment.

**Pull request merged**